__tests__/test.js

```diff
@@ -66,3 +69,27 @@ describe("test DB for requests", () => {
       });
   });
 });
+
+describe("GET /api/articles ", () => {
+  test("/api/articles returns all article in order of date", () => {
+    return request(app)
+      .get("/api/articles")
+      .then((res) => {
+        expect(res.status).toBe(200);
+        expect(res.body.length).toBeGreaterThan(0);
+        expect(res.body).toBeSorted("comments.created_at");
+        expect(res.body).toBeSorted({ descending: true });
+        res.body.forEach((topic) => {
```

Comment:
please proof read your code! each item in this array should represent a article, not a topic. Correct the parameter name here